kano/utils/audio.py

```diff
@@ -62,26 +69,15 @@ def get_volume():
     from kano.logging import logger
 
     percent = 100
-    millibel = 400
 
-    cmd = "amixer | grep %"
-    o, _, _ = run_cmd(cmd)
-    o = o.strip().split(' ')
-
-    try:
-        millibel = int(o[2])
-    except Exception:
-        msg = 'asmixer format bad for millibel, o: {}'.format(o)
-        logger.error(msg)
-        pass
+    cmd = "amixer | head -n 6 | grep -Po '(\d{1,3})(?=%)'"
```

Comment:
Are we certain that it will always be the first control? Would it not be better to search for the amixer control that we want and then select the subsequent lines?